docs/cartesian-product.adoc

```diff
@@ -113,6 +113,38 @@ To demonstrate with a table:
 | 6th test  | 4                 | "B"
 |===
 
+Alternatively, you can annotate your test method with link:range-sources.adoc[range source annotations].
```

Comment:
Note: We have to check this link (and the one below) after site deployment, as of now in the PR it points to `https://github.com/junit-pioneer/junit-pioneer/blob/d0ef6d0e6d307246a54747055866f12973d9971c/docs/range-sources.adoc` but I know that it some kind of relative link.